app/helpers/devise_helper.rb

```diff
@@ -0,0 +1,15 @@
+module DeviseHelper
+  def devise_error_messages!
```

Comment:
why is this a `!` method? from what I remember, `!` should only be appended to methods that can potentially throw an error.